prisma/schema.prisma

```diff
@@ -1,14 +1,15 @@
 datasource db {
-    provider = "sqlite"
+    provider = "postgresql"
     url      = env("DATABASE_URL")
 }
 
 generator client {
     provider = "prisma-client-js"
 }
 
-model User {
-    id    Int     @id @default(autoincrement())
-    email String  @unique
-    name  String?
+model Food {
+    id     Int    @id @default(autoincrement())
+    title  String
+    img    String
```

Comment:
Thinking about data hygiene, should `title` and/or `img` be unique?  Do any other constraint make sense here such as a maximum character count?  Or a minimum?

You can add these later, but then the data itself may need to be migrated which is less fun.

Reply:
added `@Unique` to the model food ...does that meet the minimum requirements?

Reply:
It might; I'm unfamiliar with that approach.  Can you link me to the line you changed? I'm not seeing any changes to the model at this time.